### PR TITLE
[Breadcrumbs]Changed breadcrumbs logic

### PR DESCRIPTION
**Pull request by di-shu**

### CODE REVIEW

File: src/components/Breadcrumbs.tsx [hunk 1]
```diff
@@ -1,54 +1,28 @@
 import '../styles/Breadcrumbs.scss';
-import { useEffect, useState } from 'react';
-import routing from '../config/routing';
+import { useHistory, useLocation } from "react-router";
+import {SyntheticEvent, useEffect, useState} from "react";
```
Comment (oleksandrstarov) [inline src/components/Breadcrumbs.tsx:3]: Use single quotes

File: src/components/Breadcrumbs.tsx [hunk 2]
```diff
@@ -1,54 +1,28 @@
...
+  const history = useHistory();
+  const pathname = useLocation().pathname;
 
-  const getPrevPath = () => {
-    const { name } = routing.find(({ path }): boolean =>
-      path.includes(prevPath)
-    ) || { name: '' };
-    setPrevPathName(name);
-  };
+  const handleClick = (e:SyntheticEvent ,path:string):void => {
```
Comment (oleksandrstarov) [inline src/components/Breadcrumbs.tsx:10]: fix formatting

File: src/components/Breadcrumbs.tsx [hunk 3]
```diff
@@ -1,54 +1,28 @@
...
+    e.preventDefault();
+    history.push(path);
+  }
 
   useEffect(() => {
-    setPrevPath(`/${document.referrer.split('/')[3]}`);
-    getPrevPath();
-  });
+    setPathnames(pathname.split("/").filter(item => item));
```
Comment (oleksandrstarov) [inline src/components/Breadcrumbs.tsx:16]: use single quotes

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]

**Review by Kachyr — APPROVED** [review]

File: src/components/Breadcrumbs.tsx [hunk 4]
```diff
@@ -1,54 +1,28 @@
 import '../styles/Breadcrumbs.scss';
-import { useEffect, useState } from 'react';
-import routing from '../config/routing';
+import { useHistory, useLocation } from 'react-router';
+import {SyntheticEvent, useEffect, useState} from 'react';
```
Comment (oleksandrstarov) [inline src/components/Breadcrumbs.tsx:3]: should be spaces around

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**